### PR TITLE
simpler upstream dockerfile for sync to...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

simpler upstream dockerfile for sync to midstream ([CRW-3102](https://issues.redhat.com//browse/CRW-3102))

Change-Id: I1b0508489781da4bb5b0052ba15a8881d9872b10
Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?

Refs: https://issues.redhat.com/browse/CRW-3102
Blocks: https://github.com/redhat-developer/devspaces-images/pull/299

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: build/dockerfiles/rhel.Dockerfile [hunk 1]
```diff
@@ -8,33 +8,18 @@
...
-ENV BACKEND=packages/dashboard-backend
-COPY ${BACKEND}/package.json /dashboard/${BACKEND}/
-
-WORKDIR /dashboard
-RUN npm i -g yarn
-RUN yarn install
-COPY packages/ /dashboard/packages
+COPY . /dashboard/
+WORKDIR /dashboard/
+RUN npm i -g npm yarn; yarn install
```
Comment (olexii4) [inline build/dockerfiles/rhel.Dockerfile:18]: ```... npm i -g npm yarn; ...```
->
```... npm i -g yarn; ...```
@nickboldt WDYT?

Comment (nickboldt) [reply build/dockerfiles/rhel.Dockerfile:18]: well, I figured we could also update npm at the same time, but if you prefer to only install yarn, that works for me.

Comment (olexii4) [reply build/dockerfiles/rhel.Dockerfile:18]: Thanks

**Review by olexii4 — APPROVED** [review]

**Pull request merged**